code/dataset.py

```diff
@@ -50,14 +48,19 @@ def load_dataloaders(data_dir,
                      batch_size=1024,
                      split_trainvalid=0.90,
                      t_testsplit = 5,
-                     sample_uniform_action=False):
+                     sample_uniform_slate=False):
 
     logging.info('Load data..')
-    data = torch.load(f'{data_dir}/data.pt')
-    dataset = SequentialDataset(data, sample_uniform_action)
+    with np.load(f'{data_dir}/data.npz') as data_np:
+        data = {key: torch.tensor(val) for key, val in data_np.items()}
+    dataset = SequentialDataset(data, sample_uniform_slate)
     
-    with open(f'{data_dir}/ind2val.pickle', 'rb') as handle:
-        ind2val = pickle.load(handle)
+    with open(f'{data_dir}/ind2val.json', 'rb') as handle:
+        # Use string2int object_hook found here: https://stackoverflow.com/a/54112705
+        ind2val = json.load(
+            handle, 
+            object_hook=lambda d: {int(k) if k.lstrip('-').isdigit() else k: v for k, v in d.items()}
+            )
 
     num_validusers = int(len(dataset) * (1-split_trainvalid)/2)
```

Comment:
Not completely relevant to the PR: But does it make sense to have a function parameter for `valid_pct` and `test_pct` here, instead of taking half of the remainder after selecting the train-split?

Reply:
agree! Ill make sure it doesnt change anything